python/ml4ir/base/data/kfold_relevance_dataset.py

```diff
@@ -0,0 +1,98 @@
+from typing import Optional
+from logging import Logger
+import tensorflow as tf
+
+from ml4ir.base.data.relevance_dataset import RelevanceDataset
+from ml4ir.base.features.feature_config import FeatureConfig
+from ml4ir.base.io.file_io import FileIO
+
+
+class KfoldRelevanceDataset(RelevanceDataset):
+    def __init__(
+            self,
+            data_dir: str,
+            data_format: str,
+            feature_config: FeatureConfig,
+            tfrecord_type: str,
+            file_io: FileIO,
+            max_sequence_size: int = 0,
+            batch_size: int = 128,
+            preprocessing_keys_to_fns: dict = {},
+            train_pcent_split: float = 0.8,
+            val_pcent_split: float = -1,
+            test_pcent_split: float = -1,
+            use_part_files: bool = False,
+            parse_tfrecord: bool = True,
+            logger: Optional[Logger] = None,
+            keep_additional_info: int = 0,
+            non_zero_features_only: int = 0,
+            num_folds: int = 3,
+            include_testset_in_kfold: bool = False,
+            read_data_sets: bool = False
+    ):
```

Comment:
This method needs a doc string explaining the params

Reply:
+1